pkg/server/handler/metrics/handler.go

```diff
@@ -33,45 +33,29 @@ func New(c Config) *Handler {
 
 	cou := map[string]recorder.Interface{}
 
-	{
-		nam := "teams_bridge_total"
-		cou[nam] = recorder.NewCounter(recorder.CounterConfig{
-			Des: "the total amount of bridge transactions",
-			Lab: map[string][]string{
-				"success": {"true", "false"},
-			},
-			Met: c.Met,
-			Nam: nam,
-		})
-	}
-
 	gau := map[string]recorder.Interface{}
 
-	{
-		// gauges can be registered here
-	}
-
 	his := map[string]recorder.Interface{}
 
 	{
-		nam := "teams_bridge_duration_seconds"
+		nam := "page_ready_duration_seconds"
 		his[nam] = recorder.NewHistogram(recorder.HistogramConfig{
-			Des: "the time it takes for bridge transactions to complete",
+			Des: "the time it takes for the app to show a page",
 			Lab: map[string][]string{
-				"success": {"true", "false"},
+				"page": {"root"},
```

Comment:
For now we just define the `root` page as an allowed label. If we want to do this differently we can update this later.